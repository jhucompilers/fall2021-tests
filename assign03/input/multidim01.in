PROGRAM multidim;
  TYPE Grid = ARRAY 10 OF ARRAY 10 OF INTEGER;
  VAR mygrid : Grid;

BEGIN
  mygrid[0][0] := 2;
  mygrid[1, 1] := 3;
END.
