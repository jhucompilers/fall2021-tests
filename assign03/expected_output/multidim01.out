0,TYPE,Grid,ARRAY 10 OF ARRAY 10 OF INTEGER
0,VAR,mygrid,ARRAY 10 OF ARRAY 10 OF INTEGER
